content/configuration/configuration-types.md

```diff
@@ -11,21 +11,26 @@ contributors:
 Besides exporting a single config object, there are a few more ways that cover other needs as well.
 
 
-## Exporting a function to use `--env`
+## Exporting a function to use `--env` and `argv`
```

Comment:
Maybe simplify this title to "Exporting a Function"? Then in `api/cli.md` update your link to go directly here, e.g. `/configuration/configuration-types#exporting-a-function`.